src/components/action-popover/action-popover.component.js

```diff
@@ -112,32 +130,40 @@ const ActionPopover = ({
     placement,
   };
 
+  useEffect(() => {
+    createPopper(button.current, menu.current, { placement: valueMapper() });
+  });
+
```

Comment:
Read this section https://popper.js.org/docs/v2/tutorial/#performance it looks like Popper should be destroyed after the component unmounts